tests/sles4sap/hana_cluster.pm

```diff
@@ -37,8 +57,8 @@ sub run {
 
     if (is_node(1)) {
         # Create the resource configuration
-        my $cluster_conf = 'hana_cluster.conf';
-        assert_script_run "curl -f -v " . autoinst_url . "/data/sles4sap/$cluster_conf -o /tmp/$cluster_conf";
+        my $cluster_conf = get_var('USE_SAP_HANA_SR_ANGI') ? 'angi_hana_cluster.conf' : 'hana_cluster.conf';
+        assert_script_run "curl -f -v " . autoinst_url . "/data/sles4sap/${cluster_conf} -o /tmp/${cluster_conf}";
```

Comment:
`$cluster_conf`  --> `${cluster_conf}`

I'm not sure is needed or right. This is a perl variable substituted in the command string before to be passed to `assert_script_run`